spaceward/src/components/faucet-button.tsx

```diff
@@ -25,8 +25,9 @@ function FaucetButton() {
 			disabled={loading}
 			onClick={() => getTokens()}
 			className="w-full"
+			size={"sm"}
 		>
-			GET WARD
+			Get Ward
```

Comment:
```suggestion
			Get WARD
```

when referring to the token name let's keep it capitalized